data/json/recipes/recipe_weapon.json

```diff
@@ -5634,6 +5634,112 @@
 [ "processor", 2 ]
 ]
 ]
+},
+{
+	"type" : "recipe",
+	"result": "heavy_gauss_rifle",
+	"category": "CC_WEAPON",
+	"subcategory": "CSC_WEAPON_RANGED",
+	"skill_used": "fabrication",
+	"skills_required": [["mechanics", 10], ["electronics", 9], ["computer", 2]],
+	"difficulty": 11,
+	"time": 720000,
+	"reversible": false,
+	"autolearn": true,
+	"qualities":
+		[
```

Comment:
Your indention is pretty wacky, I suspect your editor is inserting tabs or something.